target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -183,6 +183,10 @@ netgear,r7450)
 netgear,wax202)
 	ucidef_set_led_netdev "internet" "Internet" "green:net" "wan"
 	;;
+netgear,wax214v2)
+	ucidef_set_led_wlan "wifia_blue" "WiFi 2.4GHz (Blue)" "mt76-phy0" "phy0tpt"
+	ucidef_set_led_wlan "wifin_blue" "WiFi 5GHz (Blue)" "mt76-phy1" "phy1tpt"
```

Comment:
The default mt76 LED trigger is tpt so I think these configs can be removed.

Reply:
You are right, I have removed these.